apps/tests/devapps/main.go

```diff
@@ -36,9 +36,8 @@ func main() {
 	} else if exampleType == "6" {
 		// This sample does not use a serialized cache - it relies on in-memory cache by reusing the app object
 		// This works well for app tokens, because there is only 1 token per resource, per tenant.
-		acquireTokenClientCertificate()
-
-		// this time the token comes from the cache!
+		// acquireTokenClientCertificate()
+		// // this time the token comes from the cache!
```

Comment:
👀 do you intend to make this change?

Reply:
I Was testing the api on the VM so, to get these changes there pushed this, now reverted.